src/pi_sonar.cpp

```diff
@@ -164,22 +183,44 @@ int main(int argc, char *argv[])
 
   ROS_INFO("Pi Sonar node ready");
 
+  diagnostic_updater::Updater updaters[5];
```

Comment:
doesn't make sense to add this to every sonar object, instead of an external array?

Reply:
Yes, it does. I mentioned in my message that I would do it in the next update.